.travis.yml

```diff
@@ -36,14 +36,10 @@ matrix:
   include:
     # Linux {
     - os: linux
-      env: PROJECT_DIR=examples/FakeIt TOOLCHAIN=default
```

Comment:
Please comment them instead of removing. Also you can save the link to the broken build so if someone want to do fixes should be able to see what was the problem before.